storev2/rootmulti/store.go

```diff
@@ -364,14 +366,9 @@ func (rs *Store) LoadVersionAndUpgrade(version int64, upgrades *types.StoreUpgra
 			initialStores = append(initialStores, key.Name())
 		}
 	}
-	if err := rs.scStore.Initialize(initialStores); err != nil {
-		return err
-	}
-	if version > 0 {
-		_, err := rs.scStore.LoadVersion(version, false)
-		if err != nil {
-			return nil
-		}
+	rs.scStore.Initialize(initialStores)
```

Comment:
this can still error, right? if so, don't we still need to return?

Reply:
This will not error after the seidb change

Reply:
https://github.com/sei-protocol/sei-db/pull/64/files